blocks/core/ff_util/ff_util-prose/ff_util-prose.less

```diff
@@ -56,14 +56,26 @@
             color: lighten(@ff_color_control_link, 10%);
         }
     }
-    .link-no-decoration(@color:@ff_color_control_link) {
+    .link-no-dec-base(@color) {
         text-decoration: none;
         color: @color;
+    }
+    .link-no-decoration(@color: @ff_color_control_link, @color-visited: null, @color-hover: null) when ((@color-visited = null) and (@color-hover = null)) {
+        #ff_mix-util-prose > .link-no-dec-base(@color);
+        &:visited {
```

Comment:
I think this would be more useful if the repetition is avoided, like this: 
```
.link-no-dec-base(@c,@v,@h){
   text-decoration:none;
   color:@c;
   &:visited { color: @v; }
   &:active, &hover { color: @h; }
}
```
and then:
.link-no-decoration(1){
   #ff_mix-util-prose > .link-no-dec-base(@color, darken(@color, 10%), lighten(@color, 10%) );
}
.link-no-decoration(2) {
    #ff_mix-util-prose > .link-no-dec-base(@color, @color-visited, @color-hover);
}
```